packages/core/src/browser/style/tree.css

```diff
@@ -42,6 +42,10 @@
     border: none;
 }
 
+.theia-TreeContainer .ReactVirtualized__Grid__innerScrollContainer {
+    margin-bottom: calc(var(--theia-ui-padding) * 3);
+}
```

Comment:
I confirmed that the extra space works well, do we want to add it to all trees or just the explorer?

Reply:
I think we should have it for all trees, since all trees have (by default, anyway) listeners for clicks targeting their nodes but no particular tree node, and that is only guaranteed to work if we ensure that all trees have some empty space. But I don't feel very strongly, since the Navigator is the only place I'm certain it matters.